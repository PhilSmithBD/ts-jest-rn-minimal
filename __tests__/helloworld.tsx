import React, { Component } from "react";
import { Platform, StyleSheet, Text, View } from "react-native";

import "jest";
import "react-native";

import * as renderer from "react-test-renderer";

const Foo = () => {
  return (
    <View>
      <Text>Test</Text>
    </View>
  );
};

it("renders all our icons correctly", () => {
  const tree = renderer.create(<Foo />).toJSON();
  expect(tree).toMatchSnapshot();
});
