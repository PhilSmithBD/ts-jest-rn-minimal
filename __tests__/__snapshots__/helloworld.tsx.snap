// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all our icons correctly 1`] = `
<View>
  <Text>
    Test
  </Text>
</View>
`;
